docs/moosedocs.py

```diff
@@ -2,10 +2,15 @@
 import sys
 import os
 import pdb
+import re
 
 # Locate MOOSE directory
 MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..', 'moose'))
 if not os.path.exists(MOOSE_DIR):
+  RE = re.compile(r'(.*?/moose)(.*)')
+  matches = RE.search(os.getcwd())
+  MOOSE_DIR = matches.group(1)
+if not os.path.exists(MOOSE_DIR):
```

Comment:
What problem were you having with this function, is it not finding your MOOSE in a submodule or something else?